bridges/RtsBridge.php

```diff
@@ -0,0 +1,74 @@
+<?php
+class RtsBridge extends BridgeAbstract {
+
+	const NAME = 'Radio Télévision Suisse';
+	const URI = 'https://www.rts.ch/';
+	const MAINTAINER = 'imagoiq';
+	const DESCRIPTION = 'Returns newest videos from RTS';
+
+	const PARAMETERS = array(
+		'ID de l\'émission' => array(
+			'idShow' => array(
+				'name' => 'Show id',
+				'required' => true,
+				'exampleValue' => 385418,
+				'title' => 'ex. 385418 pour https://www.rts.ch/play/tv/emission/a-bon-entendeur?id=385418'
+			)
+		),
+		'ID de la section' => array(
+			'idSection' => array(
+				'name' => 'Section id',
+				'required' => true,
+				'exampleValue' => 'ce802a54-8877-49cc-acd6-8d244762829b',
+				'title' => 'ex. ce802a54-8877-49cc-acd6-8d244762829b pour https://www.rts.ch/play/tv/detail/humour?id=ce802a54-8877-49cc-acd6-8d244762829b'
+			)
+		)
+	);
+
+	public function collectData(){
+		switch($this->queriedContext) {
+		case 'ID de l\'émission':
+			$showId = $this->getInput('idShow');
+
+			$url = 'https://www.rts.ch/play/v3/api/rts/production/videos-by-show-id?showId='
+				. $showId;
+			break;
+		case 'ID de la section':
+			$sectionId = $this->getInput('idSection');
+
+			$url = 'https://www.rts.ch/play/v3/api/rts/production/media-section?sectionId='
+				. $sectionId;
+			break;
+		}
+
+		$header = array();
+ 		$input = getContents($url, $header)
+			or returnServerError('Could not request RTS.');
```

Comment:
returnServerError after getContents is redundant. Please remove it

Reply:
Fixed!